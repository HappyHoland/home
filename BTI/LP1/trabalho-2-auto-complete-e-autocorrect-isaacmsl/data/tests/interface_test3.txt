1691241 entre
1440391 sao
1177376 soa
1075440 Paulo
1027976 galinha
827090 trabalho
813125 mesmo
799622 chuveiro
768363 sal
768363 sai
753968 ainda