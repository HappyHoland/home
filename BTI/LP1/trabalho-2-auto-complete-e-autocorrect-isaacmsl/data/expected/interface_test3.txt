>>> Matches s√£o
Autocomplete | Autocorrect
sao          | sao
             | sal
             | sai
             | soa
